solr/core/src/test/org/apache/solr/core/TestCodecSupport.java

```diff
@@ -265,4 +266,27 @@ public void testCompressionModeDefault() throws IOException {
       coreContainer.unload(newCoreName);
     }
   }
+
+  public void testKnnVectorsFormat() {
+    Codec codec = h.getCore().getCodec();
+    Map<String, SchemaField> fields = h.getCore().getLatestSchema().getFields();
+    SchemaField schemaField = fields.get("vector");
+
+    PerFieldKnnVectorsFormat format = (PerFieldKnnVectorsFormat) codec.knnVectorsFormat();
+    assertEquals(
+        "Lucene95HnswVectorsFormat",
```

Comment:
minor: perhaps something like this could result in a compile-time rather than test-run breakage when `Lucene95HnswVectorsFormat` becomes `Lucene98HnswVectorsFormat` or so in a future Lucene version upgrade?
```suggestion
        Lucene95HnswVectorsFormat.class.getName(),
```

Reply:
Makes sense, thanks!

I have just replaced class.getName() with class.getSimpleName() to get rid of the full package info.